libs/kotaemon/tests/test_embedding_models.py

```diff
@@ -155,3 +157,18 @@ def test_fastembed_embeddings():
     model = FastEmbedEmbeddings()
     output = model("Hello World")
     assert_embedding_result(output)
+
+
+@skip_when_voyageai_not_installed
+@patch(
+    "voyageai.Client.embed",
+    side_effect=lambda *args, **kwargs: [[1.0, 2.1, 3.2]],
+)
+@patch(
+    "voyageai.AsyncClient.embed",
+    side_effect=lambda *args, **kwargs: [[1.0, 2.1, 3.2]],
+)
+def test_voyageai_embeddings():
+    model = VoyageAIEmbeddings()
+    output = model("Hello, world!")
+    assert_embedding_result(output)
```

Comment:
**Suggestion:** Ensure the test checks for the correct output structure.

**Existing Code:**
```Python
assert_embedding_result(output)
```

**Improved Code:**
```Python
assert isinstance(output, list) and all(isinstance(doc, DocumentWithEmbedding) for doc in output)
```

**Details:**
Ensure that the test for `VoyageAIEmbeddings` checks for the correct output structure.